### DISCUSSION TITLE
Grok weight download ?

### DISCUSSION THREAD

**Discussion by quitbiz** [xai-org/grok-1#91] [General]
Has anyone downloaded using torrent ? Can someone make it available for non torrent users to download same ?

**Comment by yxc0915** [comment]
估计得等一等，我现在正在通过torrent下载权重文件，但是速度极慢，只有3m/s左右。如果有人下载完了传到任一一个网盘就可以使用其他方式下载了

**Reply by realgump** [reply]
Try huggingface, very fast.

**Reply by caojiachen1** [reply]
https://modelscope.cn/models/cjc1887415157/grok-1/summary
我在魔搭上传了，国内应该可以满速下载

**Comment by bluevisor** [comment]
https://huggingface.co/xai-org/grok-1/tree/main

**Comment by Shreyan1** [comment]
I tried doing that to make it available for all users. But the size of this is why they probably made it to a magnet link 😂  - 
![image](https://github.com/xai-org/grok-1/assets/41754832/0c0f4ac0-bee4-40a0-a68c-4abe74369f4e)

It's better to download from the hugging face repo - https://huggingface.co/xai-org/grok-1/tree/main
